cloud/cloud_env_impl.cc

```diff
@@ -1840,6 +1843,45 @@ Status CloudEnvImpl::RollNewEpoch(const std::string& local_dbname) {
   return Status::OK();
 }
 
+Status CloudEnvImpl::UploadLocalCloudManifest(const std::string& local_dbname,
+                                              const std::string& cookie) {
+  assert(HasDestBucket());
+
+  std::string current_epoch = cloud_manifest_->GetCurrentEpoch();
+  // upload the manifest file
+  auto st = GetStorageProvider()->PutCloudObject(
+      ManifestFileWithEpoch(local_dbname, current_epoch), GetDestBucketName(),
+      ManifestFileWithEpoch(GetDestObjectPath(), current_epoch));
+  if (!st.ok()) {
+    return st;
+  }
+
+  // upload the cloud manifest file corresponds to cookie (i.e., CLOUDMANIFEST-cookie)
+  return GetStorageProvider()->PutCloudObject(
+      MakeCloudManifestFile(local_dbname, cookie), GetDestBucketName(),
+      MakeCloudManifestFile(GetDestObjectPath(), cookie));
```

Comment:
Could you also upload the cloudmanifest file in the old format (`CLOUDMANIFEST`) even when the cookie is set? That way we can quickly roll back to the world without leader follower, which doesn't have the cookie and expects `CLOUDMANIFEST` to be there.